oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/FailingDocumentStore.java

```diff
@@ -43,6 +45,8 @@
 
     private Type exceptionType = Type.GENERIC;
 
+    private List<Collection<? extends Document>> collectionIncludeList;
```

Comment:
Should be reset to null in `Fail.never()`.

Reply:
done [here](https://github.com/apache/jackrabbit-oak/pull/247/commits/20edcac5f269b088710cff473c842308df0bae06)